### PR TITLE
Remove xcconfig file, inline App Store ID

### PR DESCRIPTION
**Pull request by BPerlakiH**
Fixing #615

### CODE REVIEW

File: Support/Info.plist [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>APP_STORE_ID</key>
-	<string>$(APP_STORE_ID)</string>
+	<string>id997079563</string>
```
Comment (rgaudin) [inline Support/Info.plist:6]: How does this connects to the appstore ID ? where is it coming from ?

Comment (BPerlakiH) [reply Support/Info.plist:6]: We already had this value set, it was just not so direct.
We had: Info.plist containing key: APP_STORE_ID  -- pointing to --> ${APP_STORE_ID} which is a built time variable, which was defined in kiwix.xcconfig with the value: id997079563.
Now we have Info.plist key APP_STORE_ID with the value: id997079563.

The value itself is the published ID of the app in the App Store (generated by Apple):
https://apps.apple.com/us/app/kiwix/id997079563

We need this value to get the correct URL to redirect users to App Review (of this specific app).

Comment (BPerlakiH) [reply Support/Info.plist:6]: On the custom app side it turned out that we can do everything we want without the xcconfig files, so this extra layer can be removed.

File: Support/kiwix.xcconfig [hunk 2]
```diff
@@ -1,5 +0,0 @@
-//  Copyright © 2023 Kiwix.
-
-// Configuration settings file format documentation can be found at:
-// https://help.apple.com/xcode/#/dev745c5c974
-APP_STORE_ID = id997079563
```
Comment (BPerlakiH) [inline Support/kiwix.xcconfig:5]: @rgaudin formerly it was here

**Review by rgaudin — APPROVED** [review]
Got it ; LGTM then

**Pull request merged**